src/software/ai/navigator/path_manager/obstacle_path_manager.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+#include "software/ai/navigator/path_manager/path_manager.h"
+#include "software/util/parameter/dynamic_parameters.h"
+
+/**
+ * ObstaclePathManager uses static_obstacles to arbitrate between paths to
+ * implement the PathManager interface.
+ */
+
+class ObstaclePathManager : public PathManager
```

Comment:
i'm not sure this name is particurly distinctive. Consider the case where we add a time-based arbitrator, that also uses obstacles, just in a different way? Suggest changing to something like `VelocityObstaclePathManager`

Reply:
done